translations/en/translation.json

```diff
@@ -22,17 +22,24 @@
     },
     "globalErrors": {
       "reference": {
-        "notDefined": "There's an error due to \"{{symbol}}\" not being defined in the current scope {{location}}.\n\nIf you have defined it in your code, you should check its scope, spelling, and letter-casing (JavaScript is case-sensitive). For more:\n{{url1}}\n{{url2}}"
+        "cannotAccess": "\nError ▶️ \"{{symbol}}\" is used before declaration. Make sure you have declared the variable before using it.\n\n{{location}}\n\nFor more: {{url}}",
+        "notDefined": "\nError ▶️ \"{{symbol}}\" is not defined in the current scope {{location}}.\n\nIf you have defined it in your code, you should check its scope, spelling, and letter-casing (JavaScript is case-sensitive). For more:\n{{url1}}\n{{url2}}"
       },
       "stackSubseq": "▶️ Called from line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
       "stackTop": "▶️ Error at line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
       "syntax": {
-        "invalidToken": "There's a syntax error due to a symbol that JavaScript doesn't recognize or didn't expect at its place.\nFor more: {{url}}",
-        "unexpectedToken": "There's a syntax error due to a symbol that wasn't expected at its place.\nUsually this is due to a typo. Check the line number in the error below for anything missing/extra.\nFor more: {{url}}"
+        "badReturnOrYield": "\nSyntax Error ▶️ return lies outside of a function. Make sure you’re not missing any brackets, so that return lies inside a function.\nFor more: {{url}}",
+        "invalidToken": "\nSyntax Error ▶️ Found a symbol that JavaScript doesn't recognize or didn't expect at it's place.\nFor more: {{url}}",
+        "missingInitializer": "\nSyntax Error ▶️ A const variable is declared but not initialized. In JavaScript, an initializer for a const is required. A value must be specified in the same statement in which the variable is declared. Check the line number in the error and assign the const variable a value.\nFor more: {{url}}",
+        "redeclaredVariable": "\nSyntax Error ▶️ \"{{symbol}}\" is being redeclared. JavaScript doesn't allow declaring a variable more than once. Check the line number in error for redeclaration of the variable.\nFor more: {{url}}",
+        "unexpectedToken": "\nSyntax Error ▶️ Symbol present at a place that wasn't expected.\nUsually this is due to a typo. Check the line number in the error for anything missing/extra.\nFor more: {{url}}"
```

Comment:
"unexpectedToken" often happens when the previous line is malformed. We could suggest the user to check the previous line as well?

Reply:
@ghalestrilo that is a great idea. However, In many cases, the error is in the same line as indicated in the error message so it might create confusion.

Reply:
so I think this message is good. I am happy to defer though.

Reply:
That makes sense, let's leave it as is for now